tests/parsers/test_orca/test_orca_default.yml

```diff
@@ -471,44 +471,44 @@ output_parameters:
   zpve: 0.04908242
 relaxed_structure:
   kinds:
-  - mass: 1.00794
-    name: H
+  - mass: 12.011
+    name: C
     symbols:
-    - H
+    - C
     weights:
     - 1.0
-  - mass: 12.0107
-    name: C
+  - mass: 1.008
+    name: H
     symbols:
-    - C
+    - H
     weights:
     - 1.0
   pbc1: false
   pbc2: false
   pbc3: false
   sites:
-  - kind_name: H
+  - kind_name: C
     position:
-    - 6.902699000455
-    - 5.5216858017195
-    - 5.8053320967323
+    - 5.6454888132719
+    - 5.8099486645379
+    - 5.6434676576406
```

Comment:
Should it worry us that these positions do not match what `pymatgen` parsed? Or was the input changed somehow?

Reply:
I suspect Pymatgen was re-orienting the molecule somehow, but I will double check.